scripts/build-binary.sh

```diff
@@ -6,7 +6,7 @@ source ./scripts/test_lib.sh
 
 VER=$1
 PROJ="etcd"
-REPOSITORY="${REPOSITORY:-git@github.com:etcd-io/etcd.git}"
+REPOSITORY="${REPOSITORY:-https://github.com/etcd-io/etcd.git}"
```

Comment:
Note that also applies on other branches. Not as important for this PR.

I prefer git@github.com:etcd-io/etcd.git as during push I can depend on SSH security instead of typing in credentials. It's much safer. Best would be to take address of `origin` remote from original repo.

Reply:
I agree. But in 3.4, actually I never depend on the automatically `git push`. Instead, each time, I manually bump the version and create/push the tag. 

Let's just keep it as it was. It was working well previous. After the https://github.com/etcd-io/etcd/commit/49d05f88c31c71a48368bf20927cd8ba61a3bf1a, it broke the 3.4 release, so I tend to partially revert the change. 


We can consider to enhance it later; but I don't expect to make much big change on 3.4.